docs-source/content/userguide/managing-operators/using-the-operator/using-helm.md

```diff
@@ -468,7 +468,21 @@ Example:
 ```
 externalOperatorKey: QmFnIEF0dHJpYnV0ZXMKICAgIGZyaWVuZGx5TmFtZTogd2VibG9naWMtb3B ...
 ```
-
+##### `tokenReviewAuthentication`
+tokenReviewAuthentication, if set to `true`, specifies whether the the operator's REST API should use
```

Comment:
tokenReviewAuthentication, if set to `true`, specifies -> If set to `true`, `tokenReviewAuthentication` specifies
should use -> should use:
@lennyphan Is the following meant to be a numbered list, in other words, does `tokenReviewAuthentication` do each step in order? If so, keep it numbered. If not, then change it to a bulleted list.